fix(app): changed a bit the uri generation in mail's body (#230)
Probesys/agentj: CHANGELOG.md

### Migration notes

> - Removed `APP_DOMAIN` variable in the root `.env` file and replaced it with a more explicit `APP_URL` variable.

Reviewer: ```suggestion
- You can remove the `APP_DOMAIN` variable from the root `.env` file and you must add a `APP_URL` variable (ex. `https://agentj.example.com`).
```
